### DISCUSSION TITLE
Feedback: Work well on Dell G15 5510

### DISCUSSION THREAD

**Discussion by sfc9982** [T-Troll/alienfx-tools#169] [Show and tell]
Everything works fine on G15 5510. Include GPU limit.
FX is not supported by hardware.

[alienfx-tools](https://github.com/T-Troll/alienfx-tools) is a fantastic tool, you guys are heroes.

**Comment by T-Troll** [comment]
Thanks.

It still has issues with fan control at Windows 11 (due to it's "security") from time to time.

GPU limit supported for G15, but do nothing - ACPI BIOS function is just a stub (at least for BIOS dump i have).

PS: It's a solo project, in fact, just based on some other guy's code initially.
